assets/js/admin/admin.js

```diff
@@ -221,4 +221,60 @@
 		$(this).append(video);
 
 	});
+
+	// Change span with file name when user selects a file
```

Comment:
Inline comments should always end with a full stop.

Reply:
Resolved with 483e8ff

Reply:
The inline comment has been corrected.